schemas/repositories.json

```diff
@@ -22,6 +22,22 @@
           "type": "string",
           "pattern": "metrics\\.yaml$"
         }
+      },
+      "dependencies_url": {
+        "type": "string",
+        "format": "uri"
+      },
+      "dependencies_format": {
```

Comment:
Definitely for iOS and desktop the formats will be significantly different.  Since that's already on the roadmap, I thought we should plan for it so we don't end up in a situation where gradle is the default or something confusing...